### DISCUSSION TITLE
API Gateway Authentication and Authorization with Schema Federations

### DISCUSSION THREAD

**Discussion by hassanef** [ChilliCream/hotchocolate#4051] [Q&A]
I've implemented `Api-gateway` with `schema federations` in `.netcore`, I have some microservices like person, employee ..., and I can call all the services from `Api-gateway` with schema federations, but I have a problem about Authentication and Authorization, because I want to implement them in `Api-gateway`, and I didn't find a clear solution.

**Comment by Unstop-M** [comment]
There is no example on this topic? or abandoned the federetions project

**Comment by michaelstaib** [comment]
https://thecodeblogger.com/2021/05/25/request-header-propagation-in-net-core-web-applications/
